custom/covid/management/commands/run_all_management_command.py

```diff
@@ -52,15 +52,22 @@ def handle(self, csv_file, **options):
         pool = Pool(20)
         if options["only_inactive"]:
             for domain in domains:
-                jobs.append(pool.spawn(run_command, 'update_case_index_relationship', domain, 'contact',
-                                       location=location_ids[domain]['active']['traveler'],
-                                       inactive_location=location_ids[domain]['inactive']))
+                if 'traveler' in location_ids[domain]['active']:
```

Comment:
In case you haven't seen it, the other way this kind of code is often written is to create a `kwargs` dict, then only add `kwargs['location']` if needed, then call `jobs.append(pool.spawn(run_command, 'update_case_index_relationship', domain, 'contact', **kwargs)`